--- json-c.spec.orig	2010-04-07 22:27:33.000000000 -0700
+++ json-c.spec	2011-08-29 10:33:05.868976833 -0700
@@ -1,6 +1,6 @@
 Name:		json-c
 Version:	0.9
-Release:	1%{?dist}
+Release:	1.1%{?dist}.ness
 Summary:	A JSON implementation in C
 Group:		Development/Libraries
 License:	MIT
@@ -41,7 +41,7 @@
 done
 
 %build
-%configure --enable-shared --disable-static
+%configure --enable-shared --enable-static
 make %{?_smp_mflags}
 
 %install
@@ -65,6 +65,7 @@
 %defattr(-,root,root,-)
 %{_includedir}/json/
 %{_libdir}/libjson.so
+%{_libdir}/libjson.a
 %{_libdir}/pkgconfig/json.pc
 
 %files doc
